src/blueapi/plans/plans.py

```diff
@@ -14,7 +14,7 @@ def scan(
     detectors: List[Readable],
     axes_to_move: Mapping[str, Movable],
     spec: Spec[str],
-    metadata: Optional[Mapping[str, Any]] = None,
+    md: Optional[Mapping[str, Any]] = None,
```

Comment:
Would still prefer this is called `metadata` as it is presented to the user when they query for the plan arguments. These wrappers around `bps` and `bp` were written for two reasons:

1. add type annotations
2. make nicer argument structures with more descriptive names

Reply:
Done, I was going to argue that we don't have a count plan wrapping count, but we do. So I've made that also a little more in line